coralproject/talk
# Webpack and client plugins

## What does this PR do?

Allow plugin code from clients to be parsed by webpack.

## How do I test this PR?

Include a plugin via npm like [talk-plugin-auth-checkbox](https://www.npmjs.com/package/talk-plugin-auth-checkbox), build, notice it compiles correctly without errors.
